app/pages/foreign/ForeignPropertySummaryPage.scala

```diff
@@ -127,106 +127,106 @@ case class ForeignSummaryPage(foreignCYADiversionService: ForeignCYADiversionSer
         }
     }
 
-    val taskList =
-      Seq(
-        TaskListItem(
-          "foreign.tax",
-          foreignCYADiversionService.redirectCallToCYAIfFinished(
-            taxYear,
-            userAnswers,
-            ForeignCYADiversionService.FOREIGN_TAX,
-            Some(countryCode)
-          ) {
-            controllers.foreign.routes.ForeignIncomeTaxController.onPageLoad(taxYear, countryCode, NormalMode)
-          },
-          taskListTagForForeignTax,
-          s"foreign_property_income_tax_$countryCode"
-        ),
-        TaskListItem(
-          "foreign.income",
-          foreignCYADiversionService
-            .redirectCallToCYAIfFinished(taxYear, userAnswers, ForeignCYADiversionService.INCOME, Some(countryCode)) {
-              controllers.foreign.income.routes.ForeignPropertyIncomeStartController.onPageLoad(taxYear, countryCode)
-            },
-          taskListTagForIncome,
-          s"foreign_property_income_$countryCode"
-        )
+    val foreignTaxTaskList = TaskListItem(
+      "foreign.tax",
+      foreignCYADiversionService.redirectCallToCYAIfFinished(
+        taxYear,
+        userAnswers,
+        ForeignCYADiversionService.FOREIGN_TAX,
+        Some(countryCode)
+      ) {
+        controllers.foreign.routes.ForeignIncomeTaxController.onPageLoad(taxYear, countryCode, NormalMode)
+      },
+      taskListTagForForeignTax,
+      s"foreign_property_income_tax_$countryCode"
+    )
+    val foreignIncomeTaskList = TaskListItem(
+      "foreign.income",
+      foreignCYADiversionService.redirectCallToCYAIfFinished(
+        taxYear,
+        userAnswers,
+        ForeignCYADiversionService.INCOME,
+        Some(countryCode)
+      ) {
+        controllers.foreign.income.routes.ForeignPropertyIncomeStartController.onPageLoad(taxYear, countryCode)
+      },
+      taskListTagForIncome,
+      s"foreign_property_income_$countryCode"
+    )
+
+    val claimingAdjustmentsTaskList = TaskListItem(
```

Comment:
This should just be the adjustments tasklist. You don't need two separate task list for claiming PIA and not claiming PIA. Just use the copy function when ever you want to set PIA to false

```suggestion
       adjustmentsTaskList.copy(call =
              foreignCYADiversionService.redirectCallToCYAIfFinished(
                taxYear,
                userAnswers,
                ForeignCYADiversionService.ADJUSTMENTS,
                Some(countryCode)
              ) {
                controllers.foreign.adjustments.routes.ForeignAdjustmentsStartController
                  .onPageLoad(taxYear, countryCode, isPIA = false)
              }
```